.gitignore

```diff
@@ -0,0 +1,6 @@
+.github
+node_modules
+.hintrc
+.stylelintrc.json
+.linters.yml
```

Comment:
- [ ] You did a good job working on this milestone! 🚀 However, in order for the linters to correctly work, avoid ignoring your `.github` folder and your `linters.yml` file, this is because GitHub can identify that folder and run the respective checks in your code when you push changes! 🤓 🔧